README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+Nabilah Sammudin
+7/6/2020
+
+## Brief Biography
+
+My name is **Nabilah** and I am from **Malaysia**. I graduated from **Purdue University** back in 2017 and I am currently working as a research technician in the same lab I did my research as an undergraduate. 
```

Comment:
Hi Nabilah, this is Ben! I'm also currently working as a research technician in a lab! Hope your lab has been transitioning well during these challenging times